.github/workflows/pr.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.15.5'
+          go-version: '1.16beta1'
           stable: true
```

Comment:
the stable needs to be changed to false for beta
example in documentation https://github.com/actions/setup-go